README.md

```diff
@@ -1 +1,53 @@
-# return-of-documents
\ No newline at end of file
+# Return Of Documents
+
+## Install prerequisites
+
+Step 1 : Install Node Version Manager
+
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+        # or
+        wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+        # or
+        brew install nvm
+
+        Check the version   nvm --version
+
+Step 2 : nvm install 18.19.0
+
+Step 3: nvm use 18.19.0
+
+Step 4: npm i -g yarn
+
+Step 5: touch .env
+        then copy and paste the ROD secrets into this file
+Step 6: yarn
+
+Step 7: yarn run start:dev
+
+- [Node.js](https://nodejs.org/en/) - Tested against LTS
+- NPM (installed with Node.js) - Works with versions 2 and 3
+- [Redis server](http://redis.io/download) running on the default port
+
+
+### Database setup and integration
+
+If this is a first-time install get postgres running on the default port and setup a new, empty local database called `acrs`.
+Run [hof-rds-api](https://github.com/UKHomeOffice/hof-rds-api) locally for the acrs service. Hof-rds-api is an api that will read and write to your local database.
```

Comment:
The readme still references acrs. it's worth giving it a once-over to check it's accurate for ROD

Reply:
Changed